serial_driver/CMakeLists.txt

```diff
@@ -12,50 +11,45 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+# Co-developed by Tier IV, Inc. and Apex.AI, Inc.
 cmake_minimum_required(VERSION 3.5)
 
 project(serial_driver)
 
 ## dependencies
-find_package(ament_cmake REQUIRED)
-find_package(rclcpp REQUIRED)
-find_package(rclcpp_lifecycle REQUIRED)
-find_package(std_msgs REQUIRED)
-find_package(autoware_auto_cmake REQUIRED)
-find_package(autoware_auto_helper_functions REQUIRED)
+find_package(ament_cmake_auto REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
+
+ament_auto_find_build_dependencies()
+
 if(Boost_FOUND)
```

Comment:
Given that you're modifying the CMake file, you could get rid of this `if()`, Boost is marked as `REQUIRED`, so `Boost_FOUND` will always be true, or the configure stage of CMake will fail.

Reply:
Done. Thanks!